File: java/VerificaPostivoNegativo.java
/**
 * Programa Java define e usa uma função boolean, que indica se o número é maior ou igual a zero.
 *
 * Objetivos:
 *   - uso de variáveis, funções e com condição boolean;
 *
 * Módulo: Lógica de programação | Capítulo: XXX | Página: XXX
 * Author: Eder Magalhães
 */
public class VerificaPostivoNegativo {

    public static void main(String[] args) {
        int a = 50;
        boolean resultado = ehMaiorQueZero(a);
        
        System.out.println("O número " + a + " é positivo? " + resultado);

        int b = -5;
        System.out.println("O número " + b + " é positivo? " + ehMaiorQueZero(b));
        System.out.println("O número " + b + " é negativo? " + !ehMaiorQueZero(b));
    }

    static boolean ehMaiorQueZero(int n) {
        return n >= 0;
    }

}

File: python/verificaPostivoNegativo.py
# -*- coding: utf-8 -*-

# Programa Python define e usa uma função boolean, que indica se o número é maior ou igual a zero.
# Objetivos:
#   - uso de variáveis, funções e com condição boolean;
#
# Módulo: Lógica de programação
# Author: Eder Magalhães

def ehMaiorQueZero(n):
    return n >= 0;

a = 50;
resultado = ehMaiorQueZero(a);

print "O número ", a, " é positivo? ", resultado

b = -5;
print "O número ", b, " é positivo? ", ehMaiorQueZero(b)
print "O número ", b, " é negativo? ", not ehMaiorQueZero(b)

